templates/dj_dashboard/reservation/show.html.twig

```diff
@@ -39,8 +39,11 @@
 						<i class="bi bi-arrow-right"></i>
 					</p>
 					<p class="mt-0">
-						&nbsp{% for musicalStyle in reservation.musicalStyles %}
-							{{ musicalStyle.name }}{% if not loop.last %}, {% endif %}
+						&nbsp
+						{% for musicalStyle in reservation.musicalStyles %}
+							{{ musicalStyle.name }}
```

Comment:
peut-etre pas de rapport avec ta PR, donc ça peu etre dans un autre fix, mais tu as une fonctilon join en Twig qui fait l'equiv d'un implode, donc pas besoin de faire le not loop.last